showcase/src/index.html

```diff
@@ -30,75 +32,71 @@
 	<% } %>
 	<% } %>
 </head>
-
 <body class="">
 
-<!-- FIXME reimplement these functionalities -->
-<!-- the 'stark-init'/'stark-exit' class is set when the current route state is a child of starkAppInit/starkAppExit -->
-<!--
-<div class="stark-message stark-js-error-messages" role="alertdialog" aria-busy="true" aria-live="assertive">
-	<header><i></i></header>
-	<content>
-		<div class="stark-msg-no-js">
-			<h1>No JavaScript</h1>
-			<p>You must have javascript enabled!</p>
+	<noscript>
+		<div class="stark-no-js-warning stark-preload-ui-page"
+			 role="alertdialog"
+			 aria-busy="true"
+			 aria-live="assertive">
+			<header><i></i></header>
+			<div class="content">
+				<h1>No JavaScript</h1>
+				<p>You must have JavaScript enabled!</p>
+			</div>
 		</div>
-		<div class="stark-msg-outdated">
+	</noscript>
+
+	<div class="stark-browser-warning stark-preload-ui-page" role="alertdialog" aria-busy="true" aria-live="assertive">
+		<header><i></i></header>
+		<div class="content">
 			<h1>Browser not supported</h1>
-			<p>You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to
-				improve your experience.</p>
+			<p>You are using an <strong>outdated</strong> browser. Please
+				<a href="http://browsehappy.com"
```

Comment:
If we keep this, then we should point to the https version instead: https://browsehappy.com/